packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -274,7 +256,6 @@ export type TRenderAppOptions<AdditionalEnvironmentProperties = {}> = {
   route: string;
   disableAutomaticEntryPointRoutes: boolean;
   history: ReturnType<typeof createEnhancedHistory>;
-  adapter: typeof memoryAdapter;
```

Comment:
Kind of a breaking change in theory but I assume that we would only use this internally (if any).

So no need to bump a major.

Reply:
I don't think anybody used this. Not even we did.